firmware/arduino/chessboard/chessboard.ino

```diff
@@ -53,11 +53,17 @@ enum ErrorCode
 // Electromagnet
 #define ELECTROMAGNET 23
 #define PWM_HALF_SCALE 127 // 127 is for 50% duty cycle, 255 is for 100% duty cycle
+#define PWM_FULL_SCALE 255
```

Comment:
Can you do two things here:
- change the names to just `PWM_FULL` and `PWM_HALF`
- put `PWM FULL` before `PWM HALF` and change the comment's order respectively